### PR TITLE
Adding more Q&A

### PR DESCRIPTION
**Pull request by vriveras**

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -27,3 +27,19 @@ At this time we only support running the extension in Windows we are currently p
...
+
+To disable the conda install in WSL you can run `conda config --set auto_activate_base false` this will disable the base environment.
+
+### Do you support containers today?
+
+We are currently working on the continer support it will be enable in a future release. We currently use WSL as the location to run the pipeline and we install the pre-requisites there for you. 
+
+### Why do you need GitHub and Hugging Face credentials?
+
+We host all the project templates in GitHub and the base models are hoster in Azure or Hugging Face which requieres accounts to get access to them from the APIs.
```
Comment (ningx-ms) [inline QA.md:41]: hoster -> hosted

File: QA.md [hunk 2]
```diff
@@ -27,3 +27,19 @@ At this time we only support running the extension in Windows we are currently p
...
 
 At this time Azure GPU VMs do not support nested virtualization which is needed to run the WSL environment this will prevent the extension from working.
+
+### How can I disable the Conda auto activation from my WSL
+
+To disable the conda install in WSL you can run `conda config --set auto_activate_base false` this will disable the base environment.
+
+### Do you support containers today?
+
+We are currently working on the continer support it will be enable in a future release. We currently use WSL as the location to run the pipeline and we install the pre-requisites there for you.
```
Comment (kaysabelle) [inline QA.md:37]: >continer [](http://example.com/codeflow?start=32&length=8)

container

File: QA.md (hunk 1 above)
Comment (kaysabelle) [inline QA.md:41]: >hoster [](http://example.com/codeflow?start=68&length=6)

hosted

Comment (kaysabelle) [inline QA.md:41]: >requieres [](http://example.com/codeflow?start=106&length=9)

requires

File: QA.md [hunk 3]
```diff
@@ -27,3 +27,19 @@ At this time we only support running the extension in Windows we are currently p
...
+
+We are currently working on the continer support it will be enable in a future release. We currently use WSL as the location to run the pipeline and we install the pre-requisites there for you. 
+
+### Why do you need GitHub and Hugging Face credentials?
+
+We host all the project templates in GitHub and the base models are hoster in Azure or Hugging Face which requieres accounts to get access to them from the APIs.
+
+### I am getting an error downloading Llama2
+
+Please ensure you request access to Llama thru this form [Llama 2 sign up page](https://github.com/llama2-onnx/signup) this is needed to comply with Meta's trade compliance.
```
Comment (ningx-ms) [inline QA.md:45]: thru -> through

File: QA.md (hunk 2 above)
Comment (ningx-ms) [inline QA.md:37]: continer => container

"support it will" => "support and it will"

**Review by ningx-ms — APPROVED** [review]

**Review by ningx-ms — APPROVED** [review]

**Review by kaysabelle — APPROVED** [review]

**Pull request merged**